frontend/public/components/monitoring/alert-manager-yaml-editor.tsx

```diff
@@ -28,70 +27,59 @@ const AlertManagerYAMLEditor: React.FC<AlertManagerYAMLEditorProps> = ({
 
   const [errorMsg, setErrorMsg] = React.useState(initErrorMsg);
   const [successMsg, setSuccessMsg] = React.useState();
-  const [inProgress, setInProgress] = React.useState(false);
-  const [alertManagerYamlStr, setAlertManagerYamlStr] = React.useState(
-    !_.isEmpty(encodedAlertManagerYaml) ? Base64.decode(encodedAlertManagerYaml) : '',
-  );
+  const alertManagerYamlStr = !_.isEmpty(encodedAlertManagerYaml)
+    ? Base64.decode(encodedAlertManagerYaml)
+    : '';
 
-  const save = (e) => {
-    e.preventDefault();
-    if (_.isEmpty(alertManagerYamlStr)) {
+  const save = (yaml: string) => {
+    if (_.isEmpty(yaml)) {
       setErrorMsg('Alertmanager configuration cannot be empty.');
       setSuccessMsg('');
       return;
     }
-    setInProgress(true);
-    patchAlertManagerConfig(secret, alertManagerYamlStr).then(
+    patchAlertManagerConfig(secret, yaml).then(
       (newSecret) => {
         setSuccessMsg(
           `${newSecret.metadata.name} has been updated to version ${
             newSecret.metadata.resourceVersion
           }`,
         );
         setErrorMsg('');
-        setInProgress(false);
       },
       (err) => {
         setErrorMsg(err.message);
         setSuccessMsg('');
-        setInProgress(false);
       },
     );
   };
 
   return (
-    <div className="co-m-pane__body">
-      <form
-        className="co-m-pane__body-group co-alert-manager-yaml-form co-m-pane__form"
-        onSubmit={save}
-      >
-        <p className="co-alert-manager-yaml__explanation">
+    <>
+      <div className="co-alertmanager-yaml-editor__header">
+        <p className="help-block">
           Update this YAML to configure Routes, Receivers, Groupings and other Alertmanager settings
```

Comment:
Should have `.` at the end of the sentence.

Reply:
fixed